nextjs/src/app/[locale]/c/[public_identifier]/QRCodeModal.jsx

```diff
@@ -37,10 +27,20 @@ export default function QRCodeModal({address, chain, displayChain}) {
           }
         });
 
-        qrCode.append(window.document.getElementById('qr-wrapper'));
-      });
+export default function QRCodeModal({address, chain, displayChain}) {
+  const t = useTranslations();
+
+  useEffect(() => {
+    if(typeof window !== 'undefined') {
+      qrCode.append(window.document.getElementById('qr-wrapper'));
```

Comment:
Seems like anti pattern to access the dom directly when using react.

Reply:
agreed, will fix